pkg/collector/corechecks/cluster/orchestrator/collector_bundle.go

```diff
@@ -114,6 +114,11 @@ func (cb *CollectorBundle) Initialize() error {
 
 	for _, collector := range cb.collectors {
 		collector.Init(cb.runCfg)
+		if available := collector.IsAvailable(); !available {
```

Comment:
Good points thanks!

The collector would run indeed but doesn't log any errors just empty results because the informer isn't running. Added some logic to remove unavailable collectors 👍

Reply:
Sorry to be picky on this but instead of doing in-place updates could you more simply work on a new slice of `availableCollectors` and assign this to cb.collectors after the loop? As it's not performance sensitive let's favor greater readability 🙂

Reply:
done!